README.md

```diff
@@ -1,30 +1,29 @@
 # xk6-output-prometheus-remote
 
-[k6](https://github.com/grafana/k6) extension for publishing test-run metrics to Prometheus via Remote Write endpoint.
+This extension allows publishing test-run metrics to Prometheus via Remote Write endpoint.
```

Comment:
```suggestion
Output extension that allows publishing test-run metrics to Prometheus via Remote Write endpoint.
```

I don't like starting with `this` but maybe @heitortsergent has a better suggestion,

Reply:
I think I like the original version better tbh... But an alternative if you don't wanna start with `this`, and I agree with that feedback:

"The xk6-output-prometheus-remote extension allows you to publish test-run metrics to Prometheus via Remote Write endpoint."
(repetition isn't always bad)

Another feedback, how about including a link to the [k6 extensions page](https://k6.io/docs/extensions/)? If a user lands in this repository and doesn't know much about k6 extensions, I think that might give them the best explanation of what this is.

Reply:
https://github.com/grafana/xk6-output-prometheus-remote/pull/151/commits/21a830d8f73b1cc6f571e96226ed2cd9b9355bc1